### PR TITLE
Prepare stuff for Kusama (i.e. disable most things)

### PR DESCRIPTION
**Pull request by gavofyork**
Subsumes #363 

TODO: 
- [x] Disable eras (waiting on https://github.com/paritytech/substrate/pull/3374)

### CODE REVIEW

File: runtime/src/lib.rs [hunk 1]
```diff
@@ -435,29 +435,43 @@ construct_runtime!(
...
+		ImOnline: im_online::{Module, Call, Storage, Event, ValidateUnsigned, Config<T>},
+
+		// Governance stuff; uncallable initially.
+		Democracy: democracy::{Module, /*Call,*/ Storage, Config, Event<T>},
+		Council: collective::<Instance1>::{Module, /*Call,*/ Storage, Origin<T>, Event<T>, Config<T>},
+		TechnicalCommittee: collective::<Instance2>::{Module, /*Call,*/ Storage, Origin<T>, Event<T>, Config<T>},
+		Elections: elections::{Module, /*Call,*/ Storage, Event<T>, Config<T>},
+		TechnicalMembership: membership::<Instance1>::{Module, /*Call,*/ Storage, Event<T>, Config<T>},
+		Treasury: treasury::{Module, /*Call,*/ Storage, Event<T>},
+		// TODO: @rphmeier - we want to remove this, right?
```
Comment (rphmeier) [inline runtime/src/lib.rs]: yeah -we should remove it outright.

**Comment by gavofyork** [comment]
It's `force_era: Forcing::NotForcing`, which `== Default::default()`.

**Comment by gavofyork** [comment]
CC @jacogr ^^^

File: runtime/src/lib.rs [hunk 2]
```diff
@@ -96,6 +97,33 @@ pub fn native_version() -> NativeVersion {
...
+		&self,
+		_who: &Self::AccountId,
+		call: &Self::Call,
+		_info: DispatchInfo,
+		_len: usize,
+	) -> Result<ValidTransaction, DispatchError> {
+		match call {
+			Call::Staking(_) | Call::Claims(_) => Ok(Default::default()),
+			_ => Err(DispatchError::NoPermission),
+		}
```
Comment (kianenigma) [inline runtime/src/lib.rs]: CC especially here ^^ @jacogr 

(and acknowledging how cool this signed extension thingy is)

Question: why don't we add it to the beginning of the list then? they should execute in order afaik and if a call is not staking/claims we shouldn't check for nonce/fee etc.

**Pull request merged**